app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/data/database/GameGenieHistoryDao.kt

```diff
@@ -0,0 +1,27 @@
+package com.aptoide.android.aptoidegames.gamegenie.data.database
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.aptoide.android.aptoidegames.gamegenie.data.database.model.GameGenieHistoryEntity
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface GameGenieHistoryDao {
+
+  @Query("SELECT id from GameGenieHistory")
+  suspend fun getAllChatIds(): List<String>
+
+  @Query("SELECT * from GameGenieHistory WHERE id = :chatId")
+  suspend fun getChatById(chatId: String): List<GameGenieHistoryEntity>
+
+  @Insert(onConflict = OnConflictStrategy.REPLACE)
+  suspend fun saveChatById(gameGenieHistoryEntity: GameGenieHistoryEntity)
```

Comment:
I would rename this because we are not saving by id, we are just saving with the object so i would name it saveChat